central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -2049,7 +2048,7 @@ func (suite *PLOPDataStoreTestSuite) TestPLOPUpdateClusterIdFromBlank() {
 		Namespace:          fixtureconsts.Namespace1,
 	}
 
-	suite.True(protocompat.Equal(expectedPlopStorage, newPlopsFromDB[0]))
+	suite.Equal(expectedPlopStorage, newPlopsFromDB[0])
```

Comment:
It looks like this reverts changes from a previous commit, and breaks the pattern `assert.True(protocompat.Equal(expectedMsg, msg))`. Any reason for that ?